client/src/components/Draftail/decorators/Document.js

```diff
@@ -5,18 +5,32 @@ import Icon from '../../Icon/Icon';
 
 import TooltipEntity from '../decorators/TooltipEntity';
 
+import { STRINGS } from '../../../config/wagtailConfig';
+
 const documentIcon = <Icon name="doc-full" />;
+const missingDocumentIcon = <Icon name="warning" />;
 
 const Document = props => {
   const { entityKey, contentState } = props;
   const data = contentState.getEntity(entityKey).getData();
+  const url = data.url || null;
```

Comment:
I have changed the default value for `url` to `null`, so the "Missing document" label isn’t clickable. Same for links.